src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx

```diff
@@ -41,23 +41,16 @@ export const ArtistCVRouteFragmentContainer = createFragmentContainer(
   ArtistCVRoute,
   {
     viewer: graphql`
-      fragment ArtistCVRoute_viewer on Viewer
-        @argumentDefinitions(
-          soloShowsAtAFair: { type: "Boolean", defaultValue: false }
-          soloShowsSoloShow: { type: "Boolean", defaultValue: true }
-          groupShowsAtAFair: { type: "Boolean", defaultValue: false }
-          fairBoothsAtAFair: { type: "Boolean", defaultValue: true }
```

Comment:
i dunno about y'all, but i like my fair booths at a fair. as opposed to free-standing fair booths _not_ at a fair?!